### PR TITLE
Feature/mammals

### PR DESCRIPTION
**Pull request by andrewkalumba**
guys look through

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -42,7 +46,7 @@ <h1 class="header__page-title">Zootopia</h1>
                                 </label>
                             <ul class="sub__menu">
                                 <li>
-                                    <a href="#">Home</a>
+                                    <a href="./index.html">Home</a>
                                 </li>
```
Comment (Maatala) [inline index.html:51]: should be less spaces, and indentations at some places

File: js/Home-page.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  Array.from(sidebar.getElementsByClassName('show')).forEach(ul => {
+    ul.classList.remove('show')
+    ul.previousElementSibling.classList.remove('rotate')
+  })
+}
+
+
+
+const animals = {
+  Echidna: {
```
Comment (sabawaheed27) [inline js/Home-page.js:36]: function Animal(image, name, group, lifespan, weight, food, length, found, description) {
    this.image = image;
    this.name = name;
    this.group = group;
    this.lifespan = lifespan;
    this.weight = weight;
    this.food = food;
    this.length = length;
    this.found = found;
    this.description = description;
  }
  try like this.

File: our-team.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Zootopia</title>
+    <link rel="stylesheet" href="./styles_Index/index.css">
+    <link rel="stylesheet" href="./styles/reset.css">
+    <link rel="stylesheet" href="./styles/nav.css">
+    <link rel="stylesheet" href="./styles/footer.css">
+    <link rel="stylesheet" href="./stylesMain/about-us.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+    <script src="./js/about-us.js" defer></script>
```
Comment (sabawaheed27) [inline our-team.html:14]: You didn't link with global css?

File: our-team.html [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+            <span class="nav__bar">
+                <nav>
+                    <ul class="top__menu">
+                        <li>
+                            <input type="checkbox" id="menu-toggle" class="menu-toggle">
+                                <label for="menu-toggle" class="menu-icon">
+                                    <span></span>
+                                    <span></span>
+                                    <span></span>
+                                </label>
```
Comment (sabawaheed27) [inline our-team.html:42]: Do we really need these span or not?

File: reptiles.css [hunk 5]
```diff
@@ -0,0 +1,276 @@
+* {
+    box-sizing: border-box;
+}
+
```
Comment (sabawaheed27) [inline reptiles.css:4]: Try to link with global css.

File: mammals/mammals.html [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Zootopia</title>
+    <link rel="stylesheet" href="../styles/reset.css">
+    <link rel="stylesheet" href="../styles/nav.css">
+    <link rel="stylesheet" href="../styles/footer.css">
+    <link rel="stylesheet" href="./styles/mammals.css">
```
Comment (sabawaheed27) [inline mammals/mammals.html:11]: link global css

File: index.html (hunk 1 above)
Comment (andrewkalumba) [reply index.html:51]: gona fix it

File: our-team.html (hunk 3 above)
Comment (andrewkalumba) [reply our-team.html:14]: yes

File: our-team.html (hunk 4 above)
Comment (andrewkalumba) [reply our-team.html:42]: will look through

**Review by yevhenii-kapustian — APPROVED** [review]

**Review by SSSpac — APPROVED** [review]

**Pull request merged**